### ISSUE TITLE
Admob Rewarded video and native Banners not showing

### ISSUE THREAD

**Issue by Faehah** [unity-plugins/Unity-Admob#107]
Hi,

I am testing rewarded video ads and native Banner with admob test ads as well as a sample code which comes with the plugin, all other ads working well but the rewarded video and native banner are not showing. Please help.

Thanks

**Comment by unity-plugins** [comment]
`admob.AdmobListenerProxy.toString()' can been removed  .this not effect video and native banner.
Reward video not show ?any event or log?
Native banner need a upgrade  in the next version of the plugin.

**Comment by Faehah** [comment]
calling isRewardedVideoReady,calling loadRewardedVideo.

**Comment by Faehah** [comment]
can i add extra banners with different size in this plugin?

**Comment by unity-plugins** [comment]
Yes,you can define your different size that  admob.com supported.
